src/OVAL/probes/unix/linux/rpm-helper.c

```diff
@@ -24,8 +24,11 @@
 #include <config.h>
 #endif
 
-int rpmErrorCb (rpmlogRec rec, rpmlogCallbackData data)
+void rpmLibsPreload()
 {
-	dE("RPM: %s", rpmlogRecMessage(rec));
-	return RPMLOG_EXIT; // don't perform default logging
```

Comment:
API of the rpmlib is quite strange. We cannot replace handler by our handler. As I remember there is RPMLOG_DEFAULT - perform default logging or RPMLOG_EXIT. I understood the second value as don't perform default logging, but reality is not so. It exit everything.

We use something like:
```
if (rpmReadConfigFiles ((const char *)NULL, (const char *)NULL) != 0) {
                 dI("rpmReadConfigFiles failed: %u, %s.", errno, strerror (errno));
```

Problem is that rpm errors aren't always errors - when you scan debian - it is quite expected that we cannot read rpm database.